src/cajon.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+
+#Clase Cajon
+@dataclass(frozen=True)
+class Cajon:
+    nombre = '' #Nombre del Cajón

```

Comment:
Si vais a incluir documentación, no lo hagáis como comentarios, sino siguiendo las buenas prácticas de documentación de Python. Por favor, consultadlas y usadlas.